forward_script/src/PageInformationCollector.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Wikimedia\ForwardScript;
+
+use Mediawiki\Api\MediawikiApi;
+use Mediawiki\Api\SimpleRequest;
+use Symfony\Component\Process\Process;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Wikimedia\ForwardScript\ApplicationException;
```

Comment:
Your'e right. thanks! And I was wondering why PHPStorm always displayed
this import as unnecessary ...
Will remove it if another review makes a commit necessary.

2015-08-13 10:49 GMT+02:00 wmde-manicki <notifications@github.com>:

> In forward_script/src/PageInformationCollector.php
> <https://github.com/wmde/WikiLovesMonuments/pull/58#discussion_r36952323>:
>
> > @@ -0,0 +1,131 @@
> > +<?php
> > +
> > +namespace Wikimedia\ForwardScript;
> > +
> > +use Mediawiki\Api\MediawikiApi;
> > +use Mediawiki\Api\SimpleRequest;
> > +use Symfony\Component\Process\Process;
> > +use Symfony\Component\Process\Exception\RuntimeException;
> > +use Wikimedia\ForwardScript\ApplicationException;
>
> This import statement is not required here (not meaning it is wrong). As
> PageInformationColletor is in the same namespace as ApplicationException,
> the unqualified exception class name would be resolved properly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wmde/WikiLovesMonuments/pull/58/files#r36952323>.
>



-- 
Gabriel Birke
Software Developer

Wikimedia Deutschland e. V. | Tempelhofer Ufer 23-24 | 10963 Berlin
Phone: +49 (0)30 219 158 26-0
http://wikimedia.de

Imagine a world, in which every single human being can freely share in the
sum of all knowledge. That‘s our commitment.

Wikimedia Deutschland - Gesellschaft zur Förderung Freien Wissens e. V.
Eingetragen im Vereinsregister des Amtsgerichts Berlin-Charlottenburg unter
der Nummer 23855 B. Als gemeinnützig anerkannt durch das Finanzamt für
Körperschaften I Berlin, Steuernummer 27/681/51985.